nexus/db-queries/src/db/datastore/volume.rs

```diff
@@ -3699,7 +3781,23 @@ impl DataStore {
             paginator = p.found_batch(&haystack, &|r| r.id());
 
             for volume in haystack {
-                if volume.data().contains(&needle) {
+                let vcr: VolumeConstructionRequest =
```

Comment:
It was changed because the comparison might fail to compare due to how the strings are printed, not because they're different addresses (aka "fd00:1122:3344::101" would not equal "fd00:1122:3344:0000::101").